lib/cloudstore/cloudstore_importer.es6.js

```diff
@@ -0,0 +1,124 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+global.FOAM_FLAGS = {gcloud: true};
+require('foam2');
+require('../web_catalog/api_extractor.es6.js');
+require('../web_apis/browser.es6.js');
+require('../web_apis/web_interface.es6.js');
+require('../web_apis/browser_interface_relationship.es6.js');
+require('../web_apis/api_importer.es6.js');
+const objectGraph = require('object-graph-js').ObjectGraph;
+const fs = require('fs');
+
+foam.CLASS({
+  name: 'cloudstoreImporter',
+  package: 'org.chromium.apis.web',
+  documentation: `cloud store importer handles importing browserAPIs
+    from object graph files to a google cloud datastore.`,
+  requires: [
+    'org.chromium.apis.web.apiExtractor',
+    'org.chromium.apis.web.ApiImporter',
+    'com.google.cloud.datastore.DatastoreDAO',
+    'org.chromium.apis.web.BrowserWebInterfaceJunction',
+    'org.chromium.apis.web.Browser',
+    'org.chromium.apis.web.WebInterface',
+  ],
+  properties: [
+    {
+      name: 'extractor',
+      documentation: `Extracts interface catalog from object
+        graph obejct.`,
```

Comment:
Also, continued lines get a double-indent (ie. 4 spaces). That's throughout this PR.